composer.json

```diff
@@ -12,7 +12,9 @@
         }
     },
     "require-dev": {
-        "aws/aws-sdk-php": "^3.18"
+        "aws/aws-sdk-php": "^3.18",
```

Comment:
I see from the lock you didn't use `composer require` for this. Please in the future use it, instead of updating all dependencies, as that can be risky.